src/ngAnimate/animateCss.js

```diff
@@ -0,0 +1,914 @@
+ * the animation ends be sure to call `done(callback)` (this is the recommended way to use `$animateCss` within JavaScript-animations).
+ *
+ * The example below should put this into perspective:
+ *
+ * ```js
+ * var animation = $animateCss(element, { ... });
+ *
+ * // remember that if there is no CSS animation detected on the element
+ * // then the value returned from $animateCss will be null
+ * if (animation) {
+ *   animation.start().done(function() {
+ *     // yaay the animation is over
+ *     doneCallback();
+ *   });
+ * } else {
+ *   doneCallback();
+ * }
+ * ```
+ *
+ * @param {DOMElement} element the element that will be animated
+ * @param {object} options the animation-related options that will be applied during the animation
+ *
+ * * `event` - The DOM event (e.g. enter, leave, move). When used, a generated CSS class of `ng-EVENT` and `ng-EVENT-active` will be applied
+ * to the element during the animation. Multiple events can be provided when spaces are used as a separator. (Note that this will not perform any DOM operation.)
+ * * `easing` - The CSS easing value that will be applied to the transition or keyframe animation (or both).
+ * * `transition` - The raw CSS transition style that will be used (e.g. `1s linear all`).
+ * * `keyframe` - The raw CSS keyframe animation style that will be used (e.g. `1s my_animation linear`).
+ * * `from` - The starting CSS styles that will be applied at the start of the animation.
+ * * `to` - The ending CSS styles that will be applied across the animation via a CSS transition.
+ * * `addClass` - A space separated list of CSS classes that will be added to the element and spread across the animation.
+ * * `removeClass` - A space separated list of CSS classes that will be removed from the element and spread across the animation.
+ * * `duration` - A numeric digit representing the total duration of the transition and/or keyframe (note that a value of 1 is 1000ms). If a value of `0`
+ * is provided then the animation will be skipped entirely.
+ * * `delay` - A numeric digit representing the total delay of the transition and/or keyframe (note that a value of 1 is 1000ms). If a value of `true` is
+ * used then whatever delay value is detected from the CSS classes will be mirrored on the elements styles (e.g. by setting delay true then the style value
+ * of the element will be `transition-delay: DETECTED_VALUE`). Using `true` is useful when you want the CSS classes and inline styles to all share the same
+ * CSS delay value.
+ * * `stagger` - A numeric time value representing the delay between successively animated elements
+ * ({@link ngAnimate#css-staggering-animations Click here to learn how CSS-based staggering works in ngAnimate.})
+ * * `staggerIndex` - The numeric index representing the stagger item (e.g. a value if 5 is equal to the sixth item in the stagger; therefore when a
+ * `stagger` option value if `0.6` is used then there will be a stagger delay of `600ms`)
```

Comment:
I am not too familiar with staggering animations, but didn't you mean `0.1` (instead of `0.6`) ?